Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN GIT_VERSION=$(git describe --tags --dirty --always) && \
 FROM amazonlinux:2
 RUN yum update -y && \
     yum install -y ca-certificates && \
+    yum update curl && \
```

Comment:
why is curl needed?

Reply:
It is used by amazonlinux:2 and needs an update to fix some of the security warnings